docs/fit_spectrum.rst

```diff
@@ -2,8 +2,113 @@
 How to fit a spectrum
 #####################
 
-Details of how to fit a spectrum.
+An observed spectrum can be fit and plotted using command line scripts.
 
-From commandline.
+Fitting
+=======
 
-From code.
+A spectrum can be fit from the command line using the ``run_pahfit`` script.
+To run this script, the observed spectrum to be fit and a model pack file
+are specified.
+
+For example, to fit the Spitzer IRS SL/LL spectrum of the M101 nucleus
+(included in the data directory, from
+`Gordon et al. 2008, ApJ, 682, 336 <https://ui.adsabs.harvard.edu/#abs/2008ApJ...682..336G/abstract>`_),
+the command is:
+
+.. code-block:: console
+
+  $ run_pahfit M101_Nucleus_irs.ipac scipack_ExGal_SpitzerIRSSLLL.ipac
+
+The first argument gives the file with the observed spectrum.
+The second argument gives the file with the model pack file to be used.
+
+The ``run_pahfit`` script first looks in the current directory for the
+model pack file, then looks in the installed repository location for the
+file in the packs/ subdirectory.  Thus, a default pack can be overridden
+by copying it into the current directory and modifying it.
+
+By default, the ``run_pahfit`` script will save the output as an ASCII
+IPAC table.  In addition, the plot of the observed spectrum with the
+best fit model (including components) is saved as a pdf file.  The format
+of the output and plot files can be changed through command line arguments.
+
+Help for the possible command line options for the ``run_pahfit`` script
+can be seen by:
+
+.. code-block:: console
+
+  $ run_pahfit --help
+
+Example Output
+==============
+
+The best fit model as well as the constraints for each PAHFIT model component
+are saved in the output file.  For the example above, the output file in IPAC
+table format is below.  A value of 'null' means that parameter not used
+by that component and a value of 'nan' means that there is no constraint
+for that specified type of constraint.
+
+.. note:: Note that the output format is identical to the
+  model pack file format, allowing the output file to be used as a model
+  pack file.
+
+::
+
+  |     Name|           Form|  temp|temp_min|temp_max|temp_fixed|                  amp|amp_min|amp_max|amp_fixed|               x_0|           x_0_min|           x_0_max|x_0_fixed|                fwhm|           fwhm_min|           fwhm_max|fwhm_fixed|
```

Comment:
In Safari, each line of the table is wrapped over the next set of lines (as the width of the table is fixed so the same width of the text). In firefox and vivaldi, it looks much better with a scroll bar below the table to move through the columns.

Reply:
Not sure how to handle this one.  I'm using a standard way of showing the table, so it showing up different on different browsers is a bit out of my hands.